mkdocs.yml

```
@@ -1,17 +1,64 @@
-      python:
-        options:
-          docstring_style: numpy
-        rendering:
-          show_source: true
\ No newline at end of file
+  - search
+
+nav:
+  - Home: index.md
+  - Getting Started:
+    - Installation: getting-started/installation.md
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)